pandas/core/groupby/groupby.py

```diff
@@ -403,15 +403,23 @@ class providing the base-class of operations.
 f : function, str
     Function to apply to each group. See the Notes section below for requirements.
 
-    Can also accept a Numba JIT function with
-    ``engine='numba'`` specified.
+    Accepted inputs are:
+
+    - String function name
+    - Standard Python function
+    - User defined functions IE ``lambda x: x.sum()``
+    - Numba JIT function with ``engine='numba'`` specified.
 
+    Only passing a single function is supported with this engine.
     If the ``'numba'`` engine is chosen, the function must be
     a user defined function with ``values`` and ``index`` as the
     first and second arguments respectively in the function signature.
     Each group's index will be passed to the user defined function
     and optionally available for use.
 
+    If a string is chosen, then it needs to be the string name
+    of the groupby method you want to use.
```

Comment:
I think replacing "string name" with just "name" would be more clear here.

```suggestion
    If a string is chosen, then it needs to be the name
    of the groupby method you want to use.
```

Reply:
This has been updated here: 9223b59e6aac77dc0a12eb8585ce1e8e0c9f52d4